CONTRIBUTING.md

```diff
@@ -99,6 +99,45 @@ Tests that fail **only** in CI can be difficult and time-consuming to debug. If
 - The testing Github Action will run automatically and you will see it paused with both a Web Shell address and SSH address.
 - Copy and paste the SSH address into your terminal.
 
+## Release Hydrogen
+
+To update `@shopify/hydrogen` to new calversion, for example from `2023-01` to `2023-04` follow these steps:
+
+- Create a new branch for the version from the latest, e.g. `2023-04`.
+- Create a new changeset file updating to `major` any packages that you want released. Add any notes and guides to important updates for devs.
```

Comment:
Ah great idea, that does sound like it would work. Would just need to be sure to change not only the package.json, but also the markdown files/changelog and stuff like that.